rpcs3/Emu/Cell/Modules/cellGame.cpp

```diff
@@ -207,7 +207,11 @@ ppu_error_code cellGameBootCheck(vm::ptr<u32> type, vm::ptr<u32> attributes, vm:
 	}
 	else
 	{
-		throw fmt::exception("cellGameBootCheck(): Unknown CATEGORY: %s", category);
```

Comment:
@Nekotekina Had to pass any value, I don't think there's much difference between passing the HDD Game or Disc Game one as a hack.

@raven02 I don't think there should be a default case, because in a real PS3 the category value shouldn't be empty / unknown in any scenario, this is just a hack because something seems to be going wrong on fetching the category value from the PARAM.sfo. 

This has been happening to me with a lot of HDD Games / Disc Games for some reason, I tested a few more games on latest master and most games are returning empty categories. I believe it is an issue while reading the PARAM.SFO as I've tried some already.
Trying to boot those games with this hack works as intended.